File: Daily Flash/week 4/day 3/MySolutions/C/Program5.c
/*
Problem Statement :

Program 5: Write a Program that prints following pattern.


      4
    3 4 
  2 3 4	
1 2 3 4

*/


#include<stdio.h>

void main() {

	int num;
	
	printf("\n   ----- Pattern ----   \n\n");
	
	for(int i=0;i<4;i++) {
	
		num=0;
		for(int j=4;j>i;j--) {
			
			printf("  ");
			num++;
		}
		
		for(int k=0;k<=i;k++) {
			
			printf("%d ",num++);
		}
		
		printf("\n");
	}

	printf("\n\n");
}



/*
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 3/MySolutions/C$ cc Program5.c 
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 3/MySolutions/C$ ./a.out

   ----- Pattern ----   

        4 
      3 4 
    2 3 4 
  1 2 3 4 



*/

File: Daily Flash/week 4/day 3/MySolutions/Python/Program5.py

print("\n  ----- Pattern ------  \n")
num=0
val=1
for i in reversed(range(4)):
	num=0
	for j in range(0,i+1):
		print(end="   ")
		num=num+1
	
	for k in range(val):
		print(num,end="  ")
		num=num+1
	
	val=val+1
	print()	
			
			
'''
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 3/MySolutions/Python$ python3 Program5.py

  ----- Pattern ------  

            4  
         3  4  
      2  3  4  
   1  2  3  4  



'''			
